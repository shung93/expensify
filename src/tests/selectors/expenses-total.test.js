import selectExpensesTotal from "../../selectors/expenses-total";
import expenses from '../fixtures/expenses';

test('should return 0 if no expenses', () => {
    const res = selectExpensesTotal([]);
    expect(res).toBe(0);
});

test('should get 1 expense total correctly', () => {
    const total = selectExpensesTotal([expenses[0]]);
    expect(total).toMatchSnapshot(195);
});

test('should get expense total correctly', () => {
    const total = selectExpensesTotal(expenses);
    expect(total).toMatchSnapshot(114195);
});
