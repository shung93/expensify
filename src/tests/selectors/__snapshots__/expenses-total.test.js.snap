// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`195 1`] = `195`;

exports[`114195 1`] = `114195`;
